<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(192, 191, 188);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(140, 140, 140);
border: 2px solid black;</string>
   </property>
   <property name="text">
    <string>Create a new Project</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>531</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>210</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Models</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createbtn">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>430</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
  <widget class="QLabel" name="errortext">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>430</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Status: </string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>561</width>
     <height>411</height>
    </rect>
   </property>
   <widget class="QWidget" name="createProjPage1">
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>551</width>
       <height>411</height>
      </rect>
     </property>
     <widget class="QFrame" name="formFrame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>531</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(222, 221, 218);</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Project Name</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="projectname"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="savelocation"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Download Model?</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QCheckBox" name="downloadcheck">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="1">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="savelocbtn">
         <property name="text">
          <string>Save Location</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>531</width>
        <height>164</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="0">
        <widget class="QCheckBox" name="ssdcheck">
         <property name="text">
          <string>SSD</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QCheckBox" name="effDetCheck">
         <property name="text">
          <string>Efficent Det</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="frcnncheck">
         <property name="text">
          <string>FRCNN</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="centercheck">
         <property name="text">
          <string>CenterNet</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="frcnnrescheck">
         <property name="text">
          <string>FRCNN ResNet</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Model Type: </string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Image Size</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QCheckBox" name="smallimagecheck">
         <property name="text">
          <string>640 pix</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QCheckBox" name="largeimagecheck">
         <property name="text">
          <string>1024 pix</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QCheckBox" name="xLarge_image">
         <property name="text">
          <string>1280</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>320</y>
        <width>471</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Warning: Larger image sizes require more compute power.</string>
      </property>
     </widget>
     <widget class="QPushButton" name="newProjNextPg">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>370</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="createProjPage2">
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>321</width>
       <height>340</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="1">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Annotations</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="addAnnots">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="annots"/>
      </item>
      <item row="3" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="1">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Controlled Split</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Test Annotations</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="testAnnots"/>
      </item>
      <item row="10" column="1">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Train Annotations</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QLineEdit" name="trainAnnots"/>
      </item>
      <item row="8" column="1">
       <widget class="QPushButton" name="addTestAnnots">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QPushButton" name="addTrainAnnots">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="nextPage2">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>380</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addImages">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>49</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Images</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="imagesTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>201</width>
       <height>261</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Images</string>
      </property>
     </column>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>340</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="imageReSize"/>
      </item>
      <item>
       <widget class="QPushButton" name="resize_btn">
        <property name="text">
         <string>Resize</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="createProjPage3">
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>301</width>
       <height>181</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Enter Object Lables followed by &quot;,&quot;. </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lables"/>
      </item>
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Example: Dog, Cat, Gofer, Turtle</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="confirmLablesBtn">
        <property name="text">
         <string>Confirm</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QTableWidget" name="lablesTable">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>20</y>
       <width>181</width>
       <height>181</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Object</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="completeProjCreate">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>230</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Finish</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="createProjPage4">
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>110</y>
       <width>281</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Model Downloading...</string>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>163</y>
       <width>321</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>180</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string> Please wait for model to download and extract. </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>210</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>This window will close once complete</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
